### PR TITLE
Feat: Readme

### PR DESCRIPTION
**Pull request by gluax**
## Motivation

To have documentation for the repo.

## Changes

It starts adding a readme. Still missing some pieces; I didn't immediately write since changes will be done soon. Especially concerning the CLI overwriting config fields and CLI changes from #48.

Protip you van view the rendered readmes if you go to the branch itself! It might make reviewing easier.

Also will have a link to the seda site and, eventually, one of our images in it once the site is up.

**NOTE** All GitHub links currently point towards [sedaProtocol](https://github.com/sedaProtocol) since the readme will be public with the rebranding.

TODO:
- [x] Spell and Grammar check
- [x] Link to Seda image once the site is public.
- [x] Contributing Guidelines
  - [x] Issue Reporting
  - [x] Might need to update GitHub templates
  - [x] PR Process 
- [ ] CLI documentation(will be done in a follow up PR)
- [x] What config fields can be overwritten from the CLI

## Testing

N/A

## Related PRs and Issues

CLI docs waiting on #48.
Also, waiting for config changes for what config fields can be overwritten.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,106 @@
...
+[actions-url]: https://github.com/sedaprotocol/seda-rust/actions/workflows/push.yml+branch%3Amain
+[github-stars-badge]: https://img.shields.io/github/stars/sedaprotocol/seda-rust.svg?style=flat-square&label=github%20stars
+[github-contributors-badge]: https://img.shields.io/github/contributors/sedaprotocol/seda-rust.svg?style=flat-square
+[discord-badge]: https://img.shields.io/discord/500028886025895936.svg?logo=discord&style=flat-square
+[discord-url]: https://discord.gg/seda
+[twitter-badge]: https://img.shields.io/twitter/url/https/twitter.com/SedaProtocol.svg?style=social&label=Follow%20%40SedaProtocol
+[twitter-url]: https://twitter.com/SedaProtocol
+
+Open source Rust implementation of the
+[SEDA Protocol](https://docs.seda.xyz/seda-network/introduction/the-oracle-problem).
```
Comment (mariocao) [inline README.md:33]: In case we change the URL in the future. :)

```suggestion
[SEDA Protocol](https://docs.seda.xyz/).
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,106 @@
-# seda-rust
-Open source Rust implementation of the SEDA Protocol
+<p align="center">
+  <a href="https://seda.xyz/">
+    <img width="90%" alt="seda-rust" src="https://www.seda.xyz/images/footer/footer-image.png">
+  </a>
+</p>
 
+<h1 align="center">
+   Seda Rust
```
Comment (jamesondh) [inline README.md:8]: ~~Seda~~ SEDA

File: README.md [hunk 3]
```diff
@@ -1,3 +1,106 @@
...
+  - runtime_worker_threads(?\*) - The number of threads the node can use to spin
+    up jobs.
+  - secret_key(!\*) - Your near secret key.
+  - signer_account_id(\*) - Your near signer account id.
+- logging - All config fields related to the seda logger.
+  - log_file_path(?!\*) - The path where the log file will write.
+
+### ENV
+
+Seda configuration uses the following ENV variables if they exist.
```
Comment (jamesondh) [inline README.md:94]: ~~Seda~~ SEDA

**Review by jamesondh — CHANGES_REQUESTED** [review]

**Review by mariocao — APPROVED** [review]
Ready to go!

**Review by jamesondh — APPROVED** [review]

**Pull request merged**